packages/rabbitmq/src/rabbitmq.interfaces.ts

```diff
@@ -14,6 +14,15 @@ export interface RabbitMQExchangeConfig {
   options?: Options.AssertExchange;
 }
 
+export interface ConsumeOptions {
```

Comment:
Should this extend or pick from the official library? I wonder if they do have an internal/public interface if we shouldn't use it directly. Could you investigate that?